README.md

```diff
@@ -0,0 +1,13 @@
+### Context
+
+This is a ruby application that mimics the game hangman.  
+When the game starts, it generates a random word.  The random word is masked before displaying on screen.  The game continues until the number of attempts is exhausted or the masked word is revealed by the user successfully guessing all masked characters.
+
+### The User stories
+As a player, I want to start a game of Hangman
```

Comment:
Perhaps it's worth documenting how each (or some) of these look, eg this one could be:

```sh
$ script/hangman
```

Reply:
👍